system/jlib/jtrace.cpp

```diff
@@ -1285,16 +1285,26 @@ std::unique_ptr<opentelemetry::sdk::trace::SpanProcessor> CTraceManager::createP
     if (!exporter)
         return nullptr;
 
-    if (exportConfig->getPropBool("batch/@enabled", false))
+    if (exportConfig->getPropBool("batch/@enabled", true))
```

Comment:
It would be better if the jlog (and OS?) exporter was not batched by default.  Two possible ways of doing that relatively cleanly i) return an extra flag from createExporter or ii) use a dynamic cast to check if it is a logging class instance.